src/SSHDebugPS/ProcFSOutputParser.cs

```diff
@@ -100,9 +100,12 @@ private void ProcessLine(string line, string username)
             }
 
             // If the passed in username is empty, then treat all processes as the same user
-            bool isSameUser = string.IsNullOrWhiteSpace(username) ? true : username.Equals(procUsername, StringComparison.Ordinal);
+            bool isSameUser = string.IsNullOrWhiteSpace(systemInformation.UserName) ? true : systemInformation.UserName.Equals(procUsername, StringComparison.Ordinal);
 
-            Process process = new Process(processId, procUsername, commandLine, isSameUser);
+            // Note:
+            //    For Flags, will need to parse /proc/[pid]/stat flags.
+            //    But at this commit, flags are unused for Linux scenarios.
+            Process process = new Process(processId, systemInformation.Architecture, /* TODO: Flags */ 0, procUsername, commandLine, isSameUser);
```

Comment:
>TODO [](http://example.com/codeflow?start=88&length=4)

I don't know that this really needs a TODO comment since we don't plan to use it. Might make more sense to just update the documentation to indicate that this is absent in this case.